### PR TITLE
Support disabling kinetic scrolling for Flickable-based components

### PR DESCRIPTION
**Pull request by vSLG**
Also make horizontal scrolling available when kinetic scrolling is disabled.

### CODE REVIEW

File: src/gui/Base/HFlickable.qml [hunk 1]
```diff
@@ -4,11 +4,22 @@ import QtQuick 2.12
 import QtQuick.Controls 2.12
 
 Flickable {
-    maximumFlickVelocity: window.settings.kineticScrollingMaxSpeed
-    flickDeceleration: window.settings.kineticScrollingDeceleration
```
Comment (mirukana) [inline src/gui/Base/HFlickable.qml:8]: Why are those removed?
Looking good otherwise, thanks

Comment (vSLG) [reply src/gui/Base/HFlickable.qml]: Fixed.
